Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -254,31 +253,31 @@ private func tabData(forUser user: User?) -> TabBarItemsData {
                          isMember: isMember)
 }
 
-extension TabBarItemsData: Equatable {}
-func == (lhs: TabBarItemsData, rhs: TabBarItemsData) -> Bool {
-  return lhs.items == rhs.items &&
-         lhs.isLoggedIn == rhs.isLoggedIn &&
-         lhs.isMember == rhs.isMember
+extension TabBarItemsData: Equatable {
+  static func == (lhs: TabBarItemsData, rhs: TabBarItemsData) -> Bool {
```

Comment:
Quick lil static promotion. Bunch more of these we could update in the future. Cleans things mostly up when generic types are involved (since you can omit the generic params).